src/ir_Mitsubishi.cpp

```diff
@@ -553,11 +553,25 @@ void IRMitsubishiAC::setVane(const uint8_t position) {
   remote_state[9] |= pos;
 }
 
+// Set the requested wide-vane operation mode of the a/c unit.
+void IRMitsubishiAC::setWideVane(const uint8_t position) {
+  uint8_t pos = std::min(position, kMitsubishiAcWideVaneAuto);  // bounds check
+  pos &= 0b1111;
+  pos <<= 4;
+  remote_state[8] &= 0b00000111;  // Clear the previous setting.
+  remote_state[8] |= pos;
+}
+
 // Return the requested vane operation mode of the a/c unit.
 uint8_t IRMitsubishiAC::getVane(void) {
   return ((remote_state[9] & 0b00111000) >> 3);
 }
 
+// Return the requested wide vane operation mode of the a/c unit.
+uint8_t IRMitsubishiAC::getWideVane(void) {
+  return ((remote_state[8] & 0b11110000) >> 4);
```

Comment:
[Nitpick] similarly. the bit masking is redundant.
`return((remote_state[8] & 0b11110000) >> 4)` is the same as `return remote_state[8]  >> 4` .. for 8 bit variables.

Reply:
Yes you are right, this was just a copy from the getVane function hehe